examples/CMakeLists.txt

```
@@ -7,7 +7,8 @@ if(CMAKE_BUILD_TESTS)
   # NOTE: Example 5_Looping is built here but not run as part of the integration
   #       test suite. This is because it demonstrates 'good' versus 'bad'
   #       practice, as opposed to functionality.
-  if(NOT "${GPU_DEVICE}" STREQUAL "NONE")
+  if(NOT "${GPU_DEVICE}" STREQUAL "NONE" AND MULTI_GPU)
+    message(STATUS "Build Multi GPU integration tests")
     add_subdirectory(6_MultiGPU)
   endif()
```

General practice is to warn when tests are skipped rather than when they are included:
```suggestion
  if(NOT "${GPU_DEVICE}" STREQUAL "NONE" AND MULTI_GPU)
    add_subdirectory(6_MultiGPU)
  else()
      message(STATUS "Skipping Multi GPU integration tests")
  endif()
```

---

Addressed in [e7f1bf8](https://github.com/Cambridge-ICCS/FTorch/pull/410/commits/e7f1bf84b4e6e6a936fbda81200fb398c1de57ef)